src/host/outputStream.cpp

```diff
@@ -608,6 +608,50 @@ unsigned int ConhostInternalGetSet::GetConsoleOutputCP() const
     return ServiceLocator::LocateGlobals().getConsoleInformation().OutputCP;
 }
 
+// Routine Description:
+// - Resizes the window to the specified dimensions, in characters.
+// Arguments:
+// - width: The new width of the window, in columns
+// - height: The new height of the window, in rows
+// Return Value:
+// - True if handled successfully. False otherwise.
+bool ConhostInternalGetSet::ResizeWindow(const size_t width, const size_t height)

```

Comment:
Note that this method is essentially cut and paste from the `DispatchCommon::s_ResizeWindow` function.